fennel/client_tests/test_aggregation_optional_fields.py

```diff
@@ -0,0 +1,120 @@
+from datetime import datetime, timezone, timedelta
+from typing import Optional, List
+
+import pandas as pd
+import pytest
+
+from fennel.connectors import Webhook, source
+from fennel.datasets import (
+    LastK,
+    Count,
+    Dataset,
+    dataset,
+    field,
+    pipeline,
+    Sum,
+)
+from fennel.dtypes import Continuous
+from fennel.lib import inputs
+from fennel.testing import mock
+
+__owner__ = "nitin@fennel.ai"
+webhook = Webhook(name="fennel_webhook")
+
+
+@pytest.mark.integration
+@mock
+def test_aggregation_with_optional_columns(client):
+    @source(webhook.endpoint("Transactions"), cdc="append", disorder="14d")
+    @dataset
+    class Transactions:
+        user_id: int
+        amount: Optional[float]
+        t: datetime
+
+    @dataset(index=True)
+    class Stats:
+        user_id: int = field(key=True)
+        count_without_null: int
+        count: int
+        last_10_without_null: List[float]
+        last_10: List[Optional[float]]
+        sum: float
+        t: datetime
```

Comment:
can you extend the test to have all aggregates, min, max , avg, quantile etc

Reply:
yup added